uni/lib/json.icn

```diff
@@ -221,15 +221,16 @@ procedure jtouf(s,strict,error)
    jerror.get_err()
 end
 
-#
-# It is insane to propose a single function as the primary API for the
```

Comment:
I quite liked the self deprecating nature of this comment but, if it has to go then so be it.